index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  
+  <meta charset="UTF-8">
+  <title>Portfolio - Valentino Valkaj</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" href="style.css" />
+
+</head>
+
+<!--header-->
+<body>
+    <header id="header" class="header-scroll">
+        <nav class="nav">
+            <a href="index.html" class="my-logo">My Logo</a>
+        <img class="navbar-hamb" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACQAAAAkCAYAAADhAJiYAAAAAXNSR0IArs4c6QAAAH5JREFUWEft1UEKwCAMBVE9eenJW7pzIQ41CAmM25ASn9/aW7LVk83THIhORCGFSIDqpTL00G6C9SnGSsiBPvFSGQpGZK9dIXJbCV3UHKzfs36v/aDy+8eY7siCEdlr99qTWykhX3t67dMJUf6O1EuF+ogAfVQhhUiA6maIhF7g9golEFRtCQAAAABJRU5ErkJggg=="/>
+
+        </nav>
+
+        <!--main-->
+        <main id="main">
+            <section class="home-page">
+                <h1 class="h1">Hey There.<br />I'm Valentino.</h1>
+
+                <div class="description">
+                    <h2 class="h2-description">I'm a Software Developer</h2>
```

Comment:
- [ ] According to the project requirements, kindly replace the `<h2>` tag with a `<p>` tag.